config/data/statements.csv

```diff
@@ -0,0 +1,153 @@
+name,cohort,deadline_date,payment_date,output_fee
+
+January 2022,2021,2021-12-25,2022-1-25,true
```

Comment:
Although this is close to prod data in ECF I wonder if we can try to avoid csvs and services that read from them. Statements don't need to be exactly like prod in sandbox, and we can probably use a few random ones for each cohort, happy to chat through this